lib/network/http/utils.js

```diff
@@ -104,6 +105,20 @@ function parseRange(rangeHeader, objectSize) {
     return getByteRangeFromSpec(rangeSpec, objectSize);
 }
 
-module.exports = { parseRangeSpec,
+function checkSupportIPv6() {
+    const niList = os.networkInterfaces();
+    let supportIPv6 = false;
+    Object.keys(niList).forEach(network => {
+        supportIPv6 = niList[network].some(intfc =>
```

Comment:
It looks like you're only going to consider the latest interface because you're overwriting `supportIPv6` at each loop iteration, probably not what you wanted to do

Reply:
fixed 😸